freecodecamp - applied accessibility: fix topic typo

Ngoakor12 opened the pull request:
Related issues: [please specify]

## Description:

fix topic typo

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

ry-oc reviewed (changes requested):
Always remember when changing links we also need to update everywhere that it is referenced.

Also, getting the linter setup will help you find these issues on your own:

Linter says:
```
public/topics/free-code-camp/responsive-web-design/5-responsive-web-design-principles/index.html:                        <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"
public/syllabuses/web-dev-part-1/index.html:  <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"
public/syllabuses/web-frontend-foundations/index.html:  <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"
```

ry-oc reviewed (changes requested):
Getting there:

There are some more broken content links, remember prerequisites also need to be updated.

Maybe get someone in the team to assist with the linter setup so that you can run it yourself?

freecodecamp-topic-typofreecodecamp-topic-typo
public/topics/free-code-camp/responsive-web-design/5-responsive-web-design-principles/index.html:                        <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"
public/syllabuses/web-dev-part-1/index.html:  <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"
public/syllabuses/web-frontend-foundations/index.html:  <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"

Ngoakor12 commented:
> Getting there:
> 
> There are some more broken content links, remember prerequisites also need to be updated.
> 
> Maybe get someone in the team to assist with the linter setup so that you can run it yourself?
> 
> freecodecamp-topic-typofreecodecamp-topic-typo public/topics/free-code-camp/responsive-web-design/5-responsive-web-design-principles/index.html: <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability" public/syllabuses/web-dev-part-1/index.html: <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability" public/syllabuses/web-frontend-foundations/index.html: <span class="contentlink-missing" data="topics/free-code-camp/responsive-web-design/4-applied-accessability"

@ry-oc that seems like the same error as the one before.

How I fixed it before:
- installed the linter
- ran it and successfully reproduced the errors
- fixed the link
- ran the linter again and got no errors

To double check:
- I broke a random link
- ran the linter and it failed
- fixed it 
- and ran the linter again and it passed

Can you maybe double check that you've pulled in the new changes?
